include/utl/parser/csv.h

```diff
@@ -66,6 +67,19 @@ inline void unescape_quoted_string(std::string& arg) {
   }
 }
 
+inline cista::raw::string unescape_quoted_string(cstr& arg) {
+  cista::raw::string out;
+
+  if (arg.contains(R"("")")) {
+    std::string mut_string = arg.to_str();
```

Comment:
pkg will for each dependency

- checkout the latest commit found in any `.pkg` file, if the commit ref does not match
- if the commit ref matches, nothing will change (so you can do changes)

So the moment you create a commit in a dependency, you need to change the referenced version in the `.pkg` file to this latest commit, otherwise the previous version will be checked out.